### PR TITLE
Imported tokens components

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

We need new components for the import token modal.

image

# Changes

- New component `ImportTokenWarning` to display a warning text.
- New component  `LinkIcon`. Will be used to provide navigation to the canister page on the dashboard.
- New component `ImportTokenCanisterId`. Displays the canister ID entered by the user.

# Tests

- Added unit tests for new components.

# Todos

- [ ] Add entry to changelog (if necessary).
Not yet.

### CODE REVIEW

File: frontend/src/lib/components/accounts/ImportTokenCanisterId.svelte [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  export let label: string;
+  export let canisterId: string | undefined = undefined;
+  export let canisterLinkHref: string | undefined = undefined;
+  export let canisterIdFallback: string | undefined = undefined;
+</script>
+
+<div class="container" data-tid="import-token-canister-id-component">
+  <span data-tid="label">{label}</span>
+  <div class="canister-id">
+    {#if nonNullish(canisterId) && nonNullish(canisterLinkHref)}
```
Comment (lmuntaner) [inline frontend/src/lib/components/accounts/ImportTokenCanisterId.svelte:15]: Does that mean that withuot a canisterLinkHref there won't be a canister id?

Would it make sense to check for the link only to render the `LinkIcon` or not?

File: frontend/src/lib/components/accounts/ImportTokenWarning.svelte
Comment (lmuntaner) [inline frontend/src/lib/components/accounts/ImportTokenWarning.svelte:1]: If the message is a prop, would it make sense to rename this component to `CalloutWarning` instead?

File: frontend/src/tests/lib/components/accounts/ImportTokenCanisterId.spec.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  });
+
+  it("should not render a fallback when canister id provided", async () => {
+    const po = renderComponent(props);
+
+    expect(await po.getCanisterId().isPresent()).toEqual(true);
+    expect(await po.getCanisterIdFallback().isPresent()).toEqual(false);
+  });
+
+  it("should render fallback when canister id not provided", async () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/accounts/ImportTokenCanisterId.spec.ts:46]: Would it make sense to have the test of no fallback nor canister id for the sake of completeness?

**Review by lmuntaner — APPROVED** [review]
A few comments, but LGTM and already approving!

File: frontend/src/lib/components/accounts/ImportTokenWarning.svelte
Comment (mstrasinskis) [reply frontend/src/lib/components/accounts/ImportTokenWarning.svelte:1]: Makes sense. Renamed.

File: frontend/src/tests/lib/components/accounts/ImportTokenCanisterId.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/accounts/ImportTokenCanisterId.spec.ts:46]: Good point, added.

**Comment by mstrasinskis** [comment]
@lmuntaner, I have addressed all your comments. Thank you for the review.

**Pull request merged**